### ISSUE TITLE
Bash completion does not work correctly

### ISSUE THREAD

**Issue by hoegertn** [99designs/aws-vault#408]
The command `aws-vault --completion-bash` works perfectly for the first argument, but it does not work for the second argument.

This is correct:

```bash
$ aws-vault --completion-bash "exec"
profile1
foobar
blubb
```

But this does not work:

```bash
$ aws-vault --completion-bash "exec pr"
help
add
list
rotate
exec
remove
login
```

It should autocomplete to `profile1`

**Comment by mtibben** [comment]
Hmm perhaps this issue is in kingpin (the library providing this feature)?

**Comment by tkalus** [comment]
Definitely an issue with kingpin.  It appears to have been fixed with alecthomas/kingpin#212, but it _also_ appears that there hasn't been a v2 release to `gopkg.in` since 2017.  Kingpin.v2 seems stuck in limbo with consumers pulling from the upstream `master` or `v3-unstable`.

@mtibben Do you want a new thread/issue to cover this discussion?
